### ISSUE TITLE
Entries are listed incorrectly in some cases (causing copy problem)

### ISSUE THREAD

**Issue by nioupola** [mikyk10/alfred-keepass#1]
Hello,

First, thanks for the workflow! Almost working without issue.
I needed to edit the workflow to add a `sed` to `run keepassxc-cli to copy an attribute`. I noticed that all entries listed have a prefix (`ss/` on my case).

The first command become `q=$(echo "$ITEM_PATH" | iconv -f UTF-8-MAC -t UTF-8 | sed 's/ss\///')` instead of `q=$(echo "$ITEM_PATH" | iconv -f UTF-8-MAC -t UTF-8)`.

よろしくお願いします。

**Comment by mikyk10** [comment]
Hi tete2soja, Thanks for the comment!
Looks like the problem you encountered may be dependent on .kdbx file, but does the problem look like the attached image?

![sample1](https://user-images.githubusercontent.com/4987502/234189164-6e74e955-cfc4-4123-bd28-5813ba995d92.png)

**Comment by nioupola** [comment]
Yes. All entries are prefixed with `ss`. My file has a different name than `ss.kdbx` if it matters.

**Comment by mikyk10** [comment]
For my understanding, the database filename (`ss.kdbx`) does not matter. 
The bundled Go binary reads all .kdbx entries and removes their prefix (default: `Root`) to make an entry list.

I'll look into the way around the problem.  Thanks again for the report!

**Comment by mikyk10** [comment]
Fixed in https://github.com/mikyk10/alfred-keepass/pull/2
Thanks for the report!

The path code hard-coded stripping the first 5 characters, assuming the top-level group is always named `Root`. When the top-level group has a different name, this left an unwanted prefix (like
`ss/` in this case) on every entry. It now drops the top-level group by element, regardless of its name, so no manual sed workaround is needed.
